relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -429,6 +429,10 @@ impl DialectWithParameters {
 }
 
 impl Dialect for DialectWithParameters {
+    fn dialect(&self) -> std::any::TypeId {
```

Comment:
nit: should we `#[inline]`?

Reply:
Should we?

> Note: The rustc compiler automatically inlines functions based on internal heuristics. Incorrectly inlining functions can make the program slower, so this attribute should be used with care.

https://doc.rust-lang.org/stable/reference/attributes/codegen.html?highlight=inline#the-inline-attribute